docs/topics/api/reviewers.rst

```diff
@@ -139,3 +141,116 @@ This endpoint allows you to browse through the contents of an Add-on version.
     :>json string files.entries[].mime_category: The mime type category of this file. Can be ``image``, ``directory``, ``text`` or ``binary``.
     :>json int file.entries[].size: The size in bytes.
     :>json string file.entries[].modified: The exact time of the commit, should be equivalent with ``created``.
+
+
+-------
+Compare
+-------
+
+This endpoint allows you to compare two Add-on versions with each other.
+
+    .. note::
+        Requires authentication and the current user to have ``ReviewerTools:View``
+        permission for listed add-ons as well as ``Addons:ReviewUnlisted`` for
+        unlisted add-ons. Additionally the current user can also be the owner
+        of the add-on.
+
+.. http:get:: /api/v4/reviewers/addon/(int:addon_id)/versions/(int:version_id)/compare/(int:version_id)/
```

Comment:
This URL is not exposed actually. It should be `compare_to` and  not `compare`.

Reply:
Indeed, fixed in 7a84c2597f